src/engine/strat_engine/backstore/crypt.rs

```diff
@@ -487,8 +482,16 @@ impl CryptHandle {
         Ok(name)
     }
 
+    /// Query the Stratis metadata for the device activation name.
```

Comment:
This method looks fine, except for the header comment and error message, which look like copy/paste errors.